packages/dappmanager/src/modules/stakerConfig/set/setStakerConfig.ts

```diff
@@ -65,42 +65,39 @@ export async function setStakerConfig<T extends Network>({
     currentConsClientPkg,
     currentWeb3signerPkg
   });
-
   // Set fee recipient on db
   await setFeeRecipientOnDb(network, feeRecipient || undefined);
-
-  // EXECUTION CLIENT
-  await setExecutionClient<T>({
-    network,
-    currentExecutionClient,
-    targetExecutionClient: executionClient,
-    currentExecClientPkg
-  });
-
-  // CONSENSUS CLIENT (+ Fee recipient address + Graffiti + Checkpointsync)
-  await setConsensusClient<T>({
-    network: network,
-    feeRecipient: feeRecipient,
-    currentConsensusClient,
-    targetConsensusClient: consensusClient,
-    currentConsClientPkg
-  });
-
-  // WEB3SIGNER
-  if (enableWeb3signer !== undefined)
-    await setSignerConfig(
-      enableWeb3signer,
-      compatibleSigner.dnpName,
-      currentWeb3signerPkg
-    );
-
-  // MEV BOOST
-  await setMevBoost({
-    network,
-    mevBoost: compatibleMevBoost?.dnpName,
-    targetMevBoost: mevBoost,
-    currentMevBoostPkg
-  });
+  await Promise.all([
+    // EXECUTION CLIENT
+    await setExecutionClient<T>({
```

Comment:
Do you need to use `await` inside of `Promise.all`?

Reply:
Nice catch!